service/controller/app/v1/resource/secret/desired_test.go

```diff
@@ -267,6 +267,7 @@ func Test_Resource_GetDesiredState(t *testing.T) {
 			},
 			expectedSecret: &corev1.Secret{
 				Data: map[string][]byte{
+					// "test: app" overrides "test: catalog".
```

Comment:
The comment is because the test is a bit subtle. I don't want to use big blocks of YAML and we test `helmclient.MergeValues` in the library tests.